job_creator.php

```diff
@@ -312,16 +312,11 @@ private function createPhpunitJobs(
                 'phpunit' => true,
                 'phpunit_suite' => $suite,
             ]);
-            $matrix['include'][] = $this->createJob(1, [
-                'db' => DB_PGSQL,
-                'phpunit' => true,
-                'phpunit_suite' => $suite,
-            ]);
             // this same mysql pdo test is also created for the phpcoverage job, so only add it here if
             // not creating a phpcoverage job.
             // note: phpcoverage also runs unit tests
             if ($this->getCmsMajor() === '4' && !$this->doRunPhpCoverage($run)) {
-                $matrix['include'][] = $this->createJob(2, [
+                $matrix['include'][] = $this->createJob(1, [
```

Comment:
Is there still a job that runs with php index 1 (other than PDO which doesn't run in CMS 5)?

Reply:
There's only PHP 8.1 in CMS 5 so the index is irrelevant. We can't added unit testing for PHP 8.2 until we actually add it for real. When we do add PHP 8.2 we'll be forced to update the unit test data provider, so we can ensure that we have a nice balanced matrix then

Reply:
Kicking the can down the road lol. I don't like this approach but I won't bounce the PR back for it.